### PR TITLE
ci: merge e2e execution reports

### PR DESCRIPTION
**Pull request by a-cordier**
see https://github.com/gravitee-io/issues/issues/7906
see https://github.com/gravitee-io/issues/issues/7904
see https://github.com/gravitee-io/issues/issues/7840

🚀 CI was able to deploy the build of this PR, so you can now try it directly [here](https://apimnightlywebui24386.z6.web.core.windows.net/ci-merge-reports-run-e2e/index.html)
_Notes_: The deployed app is linked to the management API of APIM master. (Same login and password as APIM master)

---
### 🧪 End-to-End Coverage

| INSTRUCTIONS | BRANCHES |
| :----------: | :------: |
|   36% | 20%   |

A more detailed report has been uploaded to [circleci](https://output.circle-artifacts.com/output/job/623d01fc-815b-4aa4-83e7-984995e1a368/artifacts/0/gravitee-apim-e2e/jacoco/reports/index.html)

---

📚&nbsp;&nbsp;View the storybook of this branch [here](https://612657caa8e859003a8a6430-lnqfwrjiid.chromatic.com)

### CODE REVIEW

**Comment by a-cordier** [comment]
Note: there are 3 flaky test suites that prevent from getting the reports (most of the time)
  - plans-deployment
  - dynamic-properties
  - healthcheck
 
Those tests have been skip for the purpose of this review

File: .circleci/config.yml [hunk 1]
```diff
@@ -1281,73 +1278,107 @@ jobs:
...
+                        - gravitee-apim-e2e/package.json
+                        - gravitee-apim-e2e/jacoco/lib
+                        - gravitee-apim-e2e/jacoco/*.xml
             - notify-on-failure
             - store_test_results:
                 path: ~/test-results
-
+    e2e-report:
+      machine:
+        image: ubuntu-2004:202101-01
```
Comment (a-cordier) [inline .circleci/config.yml:1313]: we could spare some credits with a docker executor and a small resource class. Unfortunately, i was not able to find a docker image with ant and node in circle convenience images

**Review by gcusnieux — APPROVED** [review]

File: .circleci/config.yml (hunk 1 above)
Comment (gcusnieux) [reply .circleci/config.yml:1313]: @gaetanmaisse do you have an idea ?

Comment (gaetanmaisse) [reply .circleci/config.yml:1313]: We could either: 
 - install ant on a `cimg` image
 - create our own image based on `cimg` ones, I discussed that with @sebdevaux a few weeks ago

For the time being I would say we should go with what we have here

**Pull request merged**